unit_tests/check_common.sh

```diff
@@ -403,7 +403,7 @@ test_clamd3() {
     # Test VirusEvent feature
     run_clamdscan_fileonly $TOP/test/clam.exe
     test -f test-clamd.log || sleep 1
-    grep "Virus found: ClamAV-Test-File.UNOFFICIAL" test-clamd.log >/dev/null 2>/dev/null ||
+    grep "Virus found: ClamAV-Test-File.UNOFFICIAL in $TOP/test/clam.exe" test-clamd.log >/dev/null 2>/dev/null ||
```

Comment:
This won't pass, because $TOP may be a relative path.

Reply:
Considering that the test is going to change in the future, I updated this piece to not actually check the complete path accuracy, just match the file name at the end of a path.